index.html

```diff
@@ -35,25 +35,25 @@ <h1 id="title">Welcome to Our Survey Form</h1>
     <p id="description">Complete This Form</p>
 

```

Comment:
for having a more tidy code it's good to avoid unnecessary line breaks between elements.